docs/adr/0015-security-fixes-non-wmde-software.md

```diff
@@ -0,0 +1,75 @@
+# 15) Including and Publishing Security fixes {#adr_0015}
+
+Date: 2021-05-03
+
+## Status
+
+proposed
+
+## Context
+
+WMDE needs to decide on a process of including security fixes of non-WMDE maintained software (e.g. Mediawiki, Elasticearch, WDQS) as well as WMDE maintained software (Wikibase, WDQS-frontend) and how to publish security fix releases of Wikibase suite in such cases.
+
+Security fixes will fall under the maintenance category and will most likely be worked on by campsite, however as the running of the pipeline and publishing of docker images requires special privileges it could make sense that this step is tasked to a certain group of people or that all engineers of WMDE are given the rights to publish releases.
+
+## List of steps reporting, fixing and releasing security updates
+
+### 1. Recognizing / Reporting
+
+- Report using `security@wikimedia.org` would be forwarded to WMDE staff
+- Report using [phabricator task](https://phabricator.wikimedia.org/maniphest/task/edit/form/75/)
+- Task will only be visible to security team at first then delegated to WMDE staff.
+
+### 2. Resolving
+
+- Don't publish the solution before it's deployed in WMF production environment
+  - Dont use public repositories
+
+### 3. Deployment to production
```

Comment:
Hm, this is probably out of scope for this ADR (and it suggests that this release pipeline has anything to do with WMF production).

Reply:
Renamed the headline to "### 3. Deploy fix to WMF production environment"